Issue 109

emartinez-usgs opened the pull request:
fixes usgs/hazdev-geoserve-ws#109

jmfee-usgs commented:
- Free reference to _neicResponseview in destroy
- Configure header/styles in application
- Method comments?
- Create generic formatter as suggested by _formatMagnitude?

emartinez-usgs commented:
- Freed reference.
- Added header.
  - Will handle styles consistently once all view are complete as some "integration" step.
- Added method comments.
- Would like to refactor later when all views are complete to abstract shared code etc...
